src/libs/ReportUtils.ts

```diff
@@ -1688,7 +1695,8 @@ function getMoneyRequestReportName(report: OnyxEntry<Report>, policy: OnyxEntry<
         return Localize.translateLocal('iou.payerSpentAmount', {payer: payerName, amount: formattedAmount});
     }
 
-    if (!!report?.hasOutstandingIOU || moneyRequestTotal === 0) {
+    // The report name should show as owed also for drafts
```

Comment:
This comment is a bit weird, since it only applies to one of the conditions below.

Reply:
I will remove it from there, I agree. The conditions are quite self-explanatory so we can go without it I this

Reply:
I will remove it from there, I agree. The conditions are quite self-explanatory so we can go without it I this